examples/http/BUILD

```diff
@@ -15,6 +15,7 @@ cc_binary(
             "-DEFAULTLIB:advapi32.lib",
             "-DEFAULTLIB:crypt32.lib",
             "-DEFAULTLIB:Normaliz.lib",
+            "-DEFAULTLIB:wldap32.lib",
```

Comment:
What is this new introduced dependency for?`

Reply:
seems like there's some reliance on ldap that must have been autolinked before?

```
curl.lib(ldap.obj) : error LNK2019: unresolved external symbol __imp_ldap_search_s referenced in function _ldap_free_urldesc
```

not entirely sure since i don't have a windows machine to debug on. must be coming from a dependency somewhere

Reply:
ah actually i see, it's from curl and that's a bug where we should probably disable it.

Reply:
waiting on https://github.com/bazelbuild/bazel-central-registry/pull/2381